### DISCUSSION TITLE
Primeng OTP - Unable to change OTP values of already filled input

### DISCUSSION THREAD

**Discussion by rishanaznin00** [primefaces/community#2713] [PrimeNG] [answered]
We need to press the delete or back button if we want to change a value in the OTP field. Though the input text is highlighted and gives the user the impression that it will be updated if they press any key, that's not what happens. This is very inconvenient from the user's perspective

**Comment by qwadrox** [comment, marked as answer]
@rishanaznin00 I created an issue: https://github.com/primefaces/primeng/issues/16404

**Reply by rishanaznin00** [reply]
@qwadrox Thanks.
